.travis.yml

```diff
@@ -4,7 +4,16 @@ dist: trusty
 
```

Comment:
Add `cache: pip` to cache pip-installed dependencies.

Reply:
Fixed